packages/itwin/breakdown-trees/src/test/functionalityProviders/SpaceClipPlanesProvider.test.ts

```diff
@@ -30,7 +30,11 @@ describe("SpaceClipPlanesProvider", () => {
   const dataProviderMock = moq.Mock.ofType<IPresentationTreeDataProvider>();
 
   before(async () => {
-    await TestUtils.initializeUiFramework(connection.object);
+    try {
+      await TestUtils.initializeUiFramework(connection.object);
+    } catch (err) {
+      console.log(err);
+    }
```

Comment:
Why do we need try/catch here? This suggests that "terminate" call is missing somewhere.

Reply:
removed, was leftover from debugging